app/src/main/java/com/kickstarter/ui/fragments/BackingFragment.kt

```diff
@@ -254,6 +255,11 @@ class BackingFragment : BaseFragment<BackingFragmentViewModel.ViewModel>() {
         }
     }
 
+    override fun onStart() {
+        super.onStart()
+        this.setState(activity is BackingActivity)
```

Comment:
Just curious, how is `activity is BackingActivity` related to the fragments visibility?

Reply:
when the fragment is loaded from the activity BackingActivity, there is not animation, so the state is set to always be expanded